f5-distributed-cloud/README.md

```diff
@@ -59,10 +59,26 @@ Advanced settings include configuring batch options and TLS. You can apply limit
 	d. Select Configure in the Client Private Key field, and enter the secret in the box with type selected as Text.  
 	e. Select Blindfold, wait for the operation to complete, and click Apply.  
 
-**Step 4: Finish set up**
+**Step 4: Finish F5XC set up**
 
 - Select Save & Exit to complete creating the global log receiver. Verify that [logs][5] are received into your Datadog account.
 
+**Step 5: Create Datadog Log Facets**
+Once logs begin to arrive, it will be necessary to create [log facets][8] for data analysis and dashboard visualization.  Log facet creation is straightforward and can be accomplished from the Datadog log side panel with guidance available [here][9]. 
```

Comment:
```suggestion
Once logs begin to arrive, it will be necessary to create [log facets][8] for data analysis and dashboard visualization. Log facet creation is straightforward and can be accomplished from the Datadog log side panel with guidance available [here][9]. 
```